packages/@lwc/integration-not-karma/test/signal/untrusted/index.spec.js

```diff
@@ -0,0 +1,126 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+
+import Test from 'x/test';
+import { Signal } from 'x/signal';
+import { spyConsole } from '../../../helpers/console.js';
```

Comment:
IMO `spyConsole` doesn't add any value over just using vitest's `spyOn`. I want to remove `spyConsole` eventually, but that's a problem for later.